cdc/sink/dmlsink/mq/mq_dml_sink.go

```diff
@@ -119,35 +120,49 @@ func newDMLSink(
 
 // WriteEvents writes events to the sink.
 // This is an asynchronously and thread-safe method.
-func (s *dmlSink) WriteEvents(rows ...*dmlsink.RowChangeCallbackableEvent) error {
+func (s *dmlSink) WriteEvents(txns ...*dmlsink.CallbackableEvent[*model.SingleTableTxn]) error {
```

Comment:
Should it handle the big transaction problem ?

Reply:
big transaction is already split, like MySQL sink